feat(curriculum): add Sudoku solver project Python (#52452)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/20-upcoming-python/learn-classes-and-objects-by-building-a-sudoku-solver/6569fdc59fe1b658bc9e23a4.md

---
id: 6569fdc59fe1b658bc9e23a4
title: Step 78
challengeType: 20
dashedName: step-78
---

# --description--

> In the end, return the Board object, which represents the final state of the sudoku board after attempting to solve it.

Reviewer: ```suggestion
In the end, return your instance of the `Board` class, which represents the final state of the sudoku board after attempting to solve it.
```

# --hints--

> Simply return the gameboard object.

Reviewer: ```suggestion
You should return `gameboard` at the end of your `solve_sudoku` function.
```

```js

> assert.match(code, /return\s+gameboard/);

Reviewer: ```suggestion
({ test: () => assert.match(e.code.original["main.py"], /^(\s{4})return\s+gameboard/m) })
```
